install_config/topics/vsphere_prerequisites.adoc

```diff
@@ -16,6 +16,11 @@ Standalone ESXi is not supported.
 * vSAN, VMFS and NFS supported.
 ** vSAN support is limited to one cluster in one vCenter.
 
+[NOTE]
+====
+OpenShift 3.11 is supported and deploys on vSphere 7 clusters. Storage options vSAN, VMFS and NFS are also supported with the vSphere in-tree storage driver. 
```

Comment:
Unless you're talking about a specific subcomponent that uses "OpenShift" in its name, "{product-title}" is basically always more correct than "OpenShift." I also suggest flipping the second sentence around. 

```suggestion
{product-title} 3.11 is supported and deploys on vSphere 7 clusters. If you use the with the vSphere in-tree storage driver, vSAN, VMFS, and NFS storage options are also supported. 
```